packages/altive_lints/lint_test/lints/avoid_single_child.dart

```diff
@@ -44,12 +47,34 @@ class MyWidget extends StatelessWidget {
             Text('World'),
           ],
         ),
+        // expect_lint: avoid_single_child
+        Column(
+          children: [
+            if (random.nextBool()) const Text('Hello World'),
+          ],
+        ),
```

Comment:
If a single element is enclosed in a collection-if, it will issue a warning even if it is enclosed in a collection-if. There is no use for this, and if there is, we can just IGNORE it, right?

Reply:
Yes, I agree.